src/app/app.component.ts

```diff
@@ -65,6 +65,11 @@ export class AppComponent implements OnInit {
         this.checkForCardMatch();
       }
 
+      if (this.flippedCards.length > 2) {
```

Comment:
Thanks Mujtaba for your contribution!

This works fine, but instead of adding another if condition and reverting the state, I'd prefer to not change the state at all if the flipped cards are >= 2.

Reply:
Correct !

Reply:
If you want to modify the PR, feel free :) Otherwise, I'll just close and make the change myself